SPECS/shim-unsigned-aarch64/shim-unsigned-aarch64.spec

```diff
@@ -1,65 +1,222 @@
+%global pesign_vre 0.106-1
+%global openssl_vre 1.0.2j
+
+# For prereleases, % global prerelease rc2, and downpatch Makefile
+%if %{defined prerelease}
+%global dashpre -%{prerelease}
+%global dotpre .%{prerelease}
+%global tildepre ~%{prerelease}
+%global zdpd 0%{dotpre}.
+%endif
+
+%global efidir azurelinux
```

Comment:
Just a q: Here, for the aarch64 case, we intentionally want azurelinux?

Reply:
Both shim-unsigned-x64.spec and shim-unsigned-aarch64.spec use `%global efidir azurelinux`. Unclear if this is intentional as it matches how other distros offer shim. 

Our actual signed shim rpm (from shim.spec) still uses the old file paths `/boot/efi/EFI/BOOT/boot{arch}.efi` as described in commit https://github.com/microsoft/azurelinux/pull/10995/commits/93c34aaf82b73dba7d3187629c7eb21888cf6210